gapis/capture/capture.go

```diff
@@ -362,12 +368,13 @@ type builder struct {
 
 func newBuilder() *builder {
 	return &builder{
-		apis:     []api.API{},
-		seenAPIs: map[api.ID]struct{}{},
-		observed: interval.U64RangeList{},
-		cmds:     []api.Cmd{},
-		resIDs:   []id.ID{id.ID{}},
-		arena:    arena.New(),
+		apis:         []api.API{},
+		seenAPIs:     map[api.ID]struct{}{},
+		observed:     interval.U64RangeList{},
+		cmds:         []api.Cmd{},
+		resIDs:       []id.ID{id.ID{}},
+		arena:        arena.New(),
+		initialState: &InitialState{APIs: map[api.API]api.State{}},
```

Comment:
Maybe one day we can merge the `initialState` and `apis`. But probably too much for this CL.